demo-shell/src/app/components/cloud/task-details-cloud-demo.component.html

```diff
@@ -8,10 +8,12 @@ <h4 data-automation-id="task-details-header">Simple page to show the taskId: {{
                 [taskId]="taskId"
                 (cancelClick)="goBack()"
                 (taskCompleted)="onTaskCompleted()"
+                (taskClaimed)="onTaskClaimed()"
```

Comment:
Instead of introducing a method and using ViewChild, you can call that directly from the template, i.e. 
```
(taskClaimed)="taskHeader.ngOnInit()"
```